oss/build_whl.sh

```diff
@@ -27,7 +27,7 @@ function main() {
   write_to_bazelrc "build --cxxopt=-std=c++17"
   write_to_bazelrc "build --host_cxxopt=-std=c++17"
   write_to_bazelrc "build --experimental_repo_remote_exec"
-  write_to_bazelrc "common --check_direct_dependencies=error"
+  # write_to_bazelrc "common --check_direct_dependencies=error"
```

Comment:
This breaks downstream, especially Grain, which installs a protobuf version which is incompatible with `21.7`.